src/commands/moderation/clearwarn.ts

```diff
@@ -2,9 +2,9 @@ import { CommandCategories, CommandDefinition, createErrorEmbed } from '../index
 import { createEmbed } from '../../lib/embed';
 import Warn from '../../schemas/warn';
 
-export const clearwarns: CommandDefinition = {
-    names: ['clearwarns'],
-    description: 'Clears the specified number of warns a user has. `Arguments: <id> <count>`',
+export const clearwarn: CommandDefinition = {
+    names: ['clearwarn'],
+    description: 'Clears a warn the user has with the specified index (1-indexed). `Arguments: <id> <index>`',
```

Comment:
> description: 'Clears a warn the user has with the specified index (1-indexed). `Arguments: <id> <index>`',

After looking again, the wording and the entire command feels odd, what is the exact purpose?

Reply:
if the user has many warns (like me in fbw /s) and one warn needs to be removed, you will provide the id number of that warn to delete it. 

for example

Reply:
Ok, I wanted to originally comment, if the point of the command is to remove a singular warning, why is the description that complicated? If we show the moderators when viewing warnings the warning ID, why do we need to also tell them that they need to do math, when we can just do it in the background?

Reply:
see suggested changes below